core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/CodeWithConverter.kt

```diff
@@ -27,3 +32,19 @@ public data class CodeWithConverter(val declarations: Code, val converter: (Vari
         else -> declarations + "\n" + converter(name)
     }
 }
+
+public sealed interface CodeConverter : (VariableName) -> Code
```

Comment:
This probably needs docs to explain what it does/should do. As it's a foreign concept and essentially just a String->String, which doesn't tell much. Maybe it could steal some docs from `CodeWithConverter`